### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by gjstirling**

### CODE REVIEW

File: app/views/users/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          </li>
+          <li class="nav-item">
+            <%= link_to "Sign-out", logout_path, method: :delete, class:"nav-link" %>
+          </li>
+        <% else %>
+          <li class="nav-item">
+          <%= link_to "Login", login_path, class:"nav-link" %>
+          </li>
+          <li class="nav-item">
+          <%= link_to "sign up", signup_path, class:"nav-link" %>
```
Comment (Nicola-Carroll) [inline app/views/users/_header.html.erb]: the "Sign-out", "Login", and "sign up" on lines 20, 24 and 27 are what's visible on the UI right? If so they should follow the same format. I think they should be "Logout", "Login" and "Sign up" instead

File: config/routes.rb [hunk 2]
```diff
@@ -19,14 +19,3 @@
...
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  resources :posts do
-    resources :comments
-  end
-
-  get 'login', to: 'sessions#new'
-  post 'login', to: 'sessions#create'
-  get 'signup', to: 'users#new'
-  post 'signup', to: 'users#create'
-end
```
Comment (Nicola-Carroll) [inline config/routes.rb:32]: I'm confused what's happened here - have you deleted everything from routes apart from `root to: 'users#new'` ? If so why, that seems like a mistake?

Comment (gjstirling) [reply config/routes.rb:32]: I think you have only looked at the bottom of the file, if you view the whole file you can see there are some double routes. We just deleted the duplicates.

File: app/views/users/_header.html.erb (hunk 1 above)
Comment (gjstirling) [reply app/views/users/_header.html.erb]: Agreed

File: config/routes.rb (hunk 2 above)
Comment (Nicola-Carroll) [reply config/routes.rb:32]: AH yes! :D

**Review by Nicola-Carroll — APPROVED** [review]

**Pull request merged**